### PR TITLE
Update training to PyTorch==1.4.0

### PR DESCRIPTION
**Pull request by MARMOTatZJU**
____Major changes____

* Update all training configuration
  * learning rate multiplied by 2 in order to be compatible with PyTorch==1.4.0 & CUDA==10.1
  * Update requirements.txt
* Add tinyconv training configuration
  * _experiments/siamfcpp/train/siamfcpp_tinyconv-trn.yaml_
* Add dataloader visualization tools for SiamFC++ (SOT)
  * _demo/main/debug/visualize_siamfcpp_training_data.py_

### CODE REVIEW

File: demo/main/debug/visualize_siamfcpp_training_data.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+logger = logging.getLogger('global')
+
+def make_parser():
+    parser = argparse.ArgumentParser(description='Test')
+    parser.add_argument('--config',
+                        default='experiments/siamfcpp/train/siamfcpp_alexnet-trn.yaml',
+                        type=str,
+                        help='path to experiment configuration')
+    # parser.add_argument('--resume-from-epoch',
```
Comment (lzx1413) [inline demo/main/debug/visualize_siamfcpp_training_data.py]: this commented lines can be removed

File: demo/main/debug/visualize_siamfcpp_training_data.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+from videoanalyst.config.config import specify_task
+from videoanalyst.data import builder as dataloader_builder
+from videoanalyst.engine import builder as engine_builder
+from videoanalyst.model import builder as model_builder
+from videoanalyst.model.loss import builder as losses_builder
+from videoanalyst.optim import builder as optim_builder
+from videoanalyst.pipeline import builder as pipeline_builder
+from videoanalyst.utils import Timer, ensure_dir, complete_path_wt_root_in_cfg
+
+from videoanalyst.data.utils.visualization import show_img_FCOS
```
Comment (lzx1413) [inline demo/main/debug/visualize_siamfcpp_training_data.py]: rename show_img_FCOS to show_img_siamfcpp?
